tangelo/problem_decomposition/dmet/dmet_problem_decomposition.py

```diff
@@ -179,12 +181,12 @@ def __init__(self, opt_dict):
         # If it is a list, we verified that the length is the same.
         if not self.solvers_options:
             for solver in self.fragment_solvers:
-                if solver == "ccsd":
-                    self.solvers_options.append(default_ccsd_options)
-                elif solver == "fci":
-                    self.solvers_options.append(default_fci_options)
-                elif solver == "vqe":
+                if solver.lower() in {"ccsd", "fci", "mp2", "hf"}:
+                    self.solvers_options.append(default_classical_options)
```

Comment:
Note: this currently does not do anything, as `default_classical_options` is initialized as an empty dict and never used. L182 says "if there are no solver options passed by the user", L185 says "then we are adding nothing to that". 

- Is that the behaviour you want ?
- Are there classical options common to all these solvers ?

Just want to think a little bit about that, in case we want to put a #TODO in the code, add a comment somewhere or modify this right now.

Reply:
As of now, we are not supporting solver_options though the instantiation of the Solver classes. This was to keep the `self.solvers_options` attribute the same length vs `self.fragment_solvers`.